_posts/2022-02-02-defna-board-member-recruitment.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title:  "Join DEFNA! Board Member Recruitment"
+date: 2022-02-02 12:30:00
+categories: ""
+author_name : DEFNA
+author_url : /about
+author_avatar: defna
+show_avatar : true
+read_time : ""
+feature_image: 2022-02-02
+show_related_posts: false
+permalink: /announcements/2022/2/2/defna-board-member-recruitment/
+---
+
+
+Django Events Foundation North America (DEFNA) is looking for another board member. We have an eight member board; last year one of our board members stepped down. We are looking for a new board member, interested in growing the DjangoCon US community.
+
+Board membership takes, on average, about five hours per month performing the following functions:
+
+* Attend the monthly board meeting
+* Participate in the grant applications review process
+* Use DEFNA's social media channels to communicate about DEFNA
+* Come up with new and clever ways for DEFNA to fulfill our mission
+
+If you'd like to be considered for the board, drop us a note at hello@defna.org, and let us know:
+
+* What interests you about being on the board
+* Your current/prior community involvement (it's okay if you're new—we need everyone to participate!)
+* Any particular interests you have (grants committee, corporate filings, corporate treasurer, etc.)
+* Any ideas you have for expanding DEFNA's reach in the OSS community
+* Anything else you'd like us to know about you as a potential board member
+
+Please email your information to us by February 21st at 6:00 PM Pacific Time. We'll contact you if we need more details. We'll make our decision and reply to everyone by end-of-day March 4th.
+
+Read more about DEFNA's involvement with DjangoCon US in our [past board member recruitment post](https://www.defna.org/announcements/2017/04/03/join-us).
```

Comment:
Should we link to last year instead? https://github.com/DEFNA/defna.org/blob/main/_posts/2021-01-29-join-defna-board-member-recruitment.md

Reply:
do we need the link? there's not new information in either link. Maybe we can link instead to the About section? https://www.defna.org/about/

Reply:
I like that idea even better.